source/AI.cpp

```diff
@@ -1248,12 +1248,12 @@ bool AI::Stop(Ship &ship, Command &command, double maxSpeed, const Point *direct
 		if(direction)
 		{
 			// Time to turn from facing backwards to target:
-            double degreesFromBackwards = TO_DEG * acos(min(1., max(-1., direction->Unit().Dot(-velocity.Unit()))));
-            double turnFromBackwardsTime = degreesFromBackwards / ship.TurnRate();
-            forwardTime += turnFromBackwardsTime;
+			double degreesFromBackwards = TO_DEG * acos(min(1., max(-1., direction.Unit().Dot(-velocity.Unit()))));
+			double turnFromBackwardsTime = degreesFromBackwards / ship.TurnRate();
+			forwardTime += turnFromBackwardsTime;
 			
 			// Time to turn from facing forwards to target:
-            double degreesFromForward = TO_DEG * acos(min(1., max(-1., direction->Unit().Dot(angle.Unit()))));
+            double degreesFromForward = TO_DEG * acos(min(1., max(-1., direction.Unit().Dot(angle.Unit()))));
```

Comment:
Yeah, sorry, not sure where they're coming from, Code::Blocks is set to tabs. I'll probably try and set up a commit gate for myself :)